pkg/ensurance/executor/cpu_usage_percent.go

```diff
@@ -0,0 +1,70 @@
+package executor
+
+import (
+	"sync"
+
+	"k8s.io/klog/v2"
+
+	"github.com/gocrane/crane/pkg/ensurance/executor/podinfo"
+	"github.com/gocrane/crane/pkg/ensurance/executor/sort"
+	"github.com/gocrane/crane/pkg/metrics"
+	"github.com/gocrane/crane/pkg/utils"
+)
+
+func init() {
+	registerMetricMap(cpuUsagePercent)
+}
+
+var cpuUsagePercent = metric{
+	Name:           CpuUsagePercent,
+	ActionPriority: 5,
+	Sortable:       true,
+	SortFunc:       sort.CpuUsageSort,
+
+	Throttleable:       false,
+	ThrottleQuantified: false,
+	ThrottleFunc:       nil,
+	RestoreFunc:        nil,
+
+	Evictable:       true,
+	EvictQuantified: true,
+	EvictFunc:       cpuUsagePercentEvictPod,
+}
+
+func cpuUsagePercentEvictPod(wg *sync.WaitGroup, ctx *ExecuteContext, index int, totalReleasedResource *ReleaseResource, EvictPods EvictPods) (errPodKeys []string, released ReleaseResource) {
+	wg.Add(1)
```

Comment:
seems you did not call wg.wait in main thread

Reply:
We call it in crane/pkg/ensurance/executor/evict.go:113